Northstar.CustomServers/mod/scripts/vscripts/messages_server.nut

```diff
@@ -0,0 +1,16 @@
+untyped
+globalize_all_functions
```

Comment:
```suggestion
global function MessageLoop_Init
```
There's no point in making `MessageLoop_Thread` global

Reply:
This is a bit inconsequential